docs/source-fabric/guide/lightning_module.rst

```diff
@@ -7,7 +7,7 @@ Organize Your Code
 Any raw PyTorch can be converted to Fabric with zero refactoring required, giving maximum flexibility in how you want to organize your projects.
 
 However, when developing a project in a team or sharing the code publicly, it can be beneficial to conform to a standard format of how core pieces of the code are organized.
-This is what the :doc:`LightningModule <../../common/lightning_module>` was made for!
+This is what the LightningModule was made for!
```

Comment:
Can't we have links between top-level projects?

Reply:
We can have later html ref, but note that we are generating each dos as stand-alone no jointly, see laos: https://github.com/Lightning-AI/lightning/pull/16742#discussion_r1119790964

Reply:
I suggest we put the https url here to lightning stable docs

Reply:
spoke with @justusschock and we bay link external docs similar as we refer to PyTorch or NumPy... :otter:

Reply:
```suggestion
This is what the :class:`pytorch_lightning.core.module.LightningModule` was made for!
```